test/spec/modules/taboolaBidAdapter_spec.js

```diff
@@ -275,6 +275,26 @@ describe('Taboola Adapter', function () {
       expect(resData.tmax).to.equal(500);
     });
 
+    it('should pass bidder timeout as a string', function () {
```

Comment:
test name is incorrect, should be "should pass tmax as int - tmax received as string"